format.py

```diff
@@ -175,10 +185,18 @@ def main():
                 f"Error: neither clang-apply-replacements nor clang-apply-replacements-{CLANG_VER} found (required to use -j)"
             )
 
-    if args.files:
-        files = args.files
+    if args.files or args.files_list:
+        files = []
+        files.extend(args.files)
+        if args.files_list:
+            files.extend(Path(args.files_list).read_text().split())
+        files = list(files)
         extra_files = []
+        if args.verbose:
+            print("Formatting specific files:", len(files), files)
```

Comment:
Nit: I think something like `f"Formatting {len(files)} files: ' '.join(files)"` would be more friendly, but I doubt many people will actually read this

Reply:
yeah this is meant for debugging when this implementation explodes, I don't really care how it looks